docs/serving/configuring-autoscaling.md

```diff
@@ -1,24 +1,26 @@
 ---
-title: "Configuring the Autoscaler"
+title: "Configuring autoscaling "
 weight: 10
 type: "docs"
+aliases:
+- /docs/serving/configuring-the-autoscaler/
 ---
 
-Since Knative v0.2, per revision autoscalers have been replaced by a single
-shared autoscaler. This is, by default, the Knative Pod Autoscaler (KPA), which
+Knative uses a single shared autoscaler. This is, by default, the Knative Pod Autoscaler (KPA), which
 provides fast, request-based autoscaling capabilities out of the box.
+You can also configure Knative to use Horizontal Pod Autoscaler (HPA), or use your own autoscaler, by implementing the Pod Autoscaler custom resource.
```

Comment:
Per the other comment the user is not "implementing" or creating a CR (thats pre existing). I think the accurate task here is to "create a reconciler" for the existing CR.